### ISSUE TITLE
Missing links on the website

### ISSUE THREAD

**Issue by llunaCreixent** [CirclesUBI/circles-handbook#35]
There's no way to navigate to the page [`/developers/tutorials`](https://handbook.joincircles.net/docs/developers/tutorials/) through clicks from the main page.

**Comment by louilinn** [comment]
Is this still an issue?

**Comment by llunaCreixent** [comment]
We might want to go to `https://handbook.joincircles.net/docs/developers/tutorials/` when clicking in **Tutorials**.

![menu](https://user-images.githubusercontent.com/25456102/127147039-7b40e674-9e76-4627-8c79-57e762be3eeb.jpg)

**Comment by louilinn** [comment]
Yes I agree but I think @adzialocha changed to a dropdown meny (which is nice I think!) and as far as I can tell, you can't do both. Do you know more @adzialocha?
